provider.go

```diff
@@ -340,16 +351,37 @@ func (p *Provider) putMedia(baseTimecode chan uint64, ch chan ebml.Block, chTag
 			errFlush = fmt.Errorf("flushing buffer: %w", err)
 		}
 	}()
-	errPutMedia := p.putMediaRaw(r, chResp, opts)
+
+	chRespRaw := make(chan FragmentEvent)
+	var wgResp sync.WaitGroup
+	defer wgResp.Wait()
+	wgResp.Add(1)
+	go func() {
+		defer wgResp.Done()
+		for fe := range chRespRaw {
+			if fe.ErrorId == INVALID_MKV_DATA && conn.numBlock() == 0 {
+				// Ignore INVALID_MKV_DATA due to zero Block segment.
+				continue
+			}
```

Comment:
Why the filter is needed?
In my understand, `chResp` is to tell results of each PutMedia to client.
It doesn't affect internal error handling and client might want to know INVALID_MKV_DATA error occurred. So INVALID_MKV_DATA should not be filtered here, I think.

Reply:
This error is occurred when kinesisvideomanager prepared a new connection but the connection was not actually used. (Fragment is not registered to KVS)
I think INVALID_MKV_DATA should be returned only if mkv is actually wrong, otherwise users can't notice real INVALID_MKV_DATA error caused by a bug in kinesisvideomanager or ebml-go .

Reply:
Thank you for the explanation. It makes sense 😺